src/ui/MessageDialog.vala

```diff
@@ -33,7 +33,8 @@ namespace Ev3devKit.Ui {
     public class MessageDialog : Ev3devKit.Ui.Dialog {
         Scroll vscroll;
         Label title_label;
-        Label message_label;
+
+        public Label message_label { get; construct; }
```

Comment:
What I have done on some other widget is that instead of exposing the internal widget, I just exposed font and color as optional parameters in the constructor. I think I did this on the Button widget.

Part of what I was going for with the MessageDialog was for all messages to look pretty much the same. What sorts of styling changes do you intend to make using this property?